src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -555,6 +555,10 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
           return (deviceMediaTrackConstraints.deviceId as ConstrainDOMStringParameters)
             .exact as string;
         }
+      } else {
```

Comment:
Do we need to do the same in getActiveDeviceId?

Reply:
Good catch, and I see Kyu's comments below, I'll update accodingly